### DISCUSSION TITLE
Scrape paid content from expired/non-active users

### DISCUSSION THREAD

**Discussion by DylanRick** [DIGITALCRIMINALS/OnlyFans#302] [Q&A]
Is it possible to scrape paid content from a User that’s no longer active or who’s subscription has expired?

**Comment by RayRho93** [comment]
It started to work for me when I had these values in the OnlyFans section in the config file:
"auto_model_choice": true,
"auto_media_choice": "All",

Just in case these two changes don't work for you, here is my OnlyFans section in the config file:
"onlyfans": {
      "settings": {
        "auto_profile_choice": "default",
        "auto_model_choice": true,
        "auto_api_choice": true,
        "auto_media_choice": "All",
        "browser": {
          "auth": true
        },
        "jobs": {
          "scrape": {
            "subscriptions": true,
            "paid_content": true
          },
          "metadata": {
            "posts": false,
            "comments": false
          }
        },
        "download_directories": [
          "{Your_Download_Path}"
        ],
        "file_directory_format": "{site_name}/{model_username}/{api_type}/{value}/{media_type}",
        "filename_format": "{filename}.{ext}",
        "metadata_directories": [
          ".sites"
        ],
        "metadata_directory_format": "{site_name}/{model_username}/Metadata",
        "delete_legacy_metadata": false,
        "text_length": 255,
        "video_quality": "source",
        "overwrite_files": false,
        "date_format": "%m-%d-%Y",
        "ignored_keywords": [],
        "ignore_type": "",
        "blacklists": [],
        "webhook": true
      }
    },
